client/terraform_cli_test.go

```diff
@@ -14,11 +14,14 @@ import (
 
 func NewTestTerraformCLI(config *TerraformCLIConfig, tfMock *mocks.TerraformExec) *TerraformCLI {
 	if tfMock == nil {
+		tfvars := tfexec.VarFile("terraform.tfvars")
+		ptfvars := tfexec.VarFile("providers.tfvars")
+
 		m := new(mocks.TerraformExec)
 		m.On("SetEnv", mock.Anything).Return(nil)
 		m.On("Init", mock.Anything).Return(nil)
-		m.On("Apply", mock.Anything, mock.Anything).Return(nil)
-		m.On("Plan", mock.Anything, mock.Anything).Return(true, nil)
+		m.On("Apply", mock.Anything, tfvars, ptfvars).Return(nil)
+		m.On("Plan", mock.Anything, tfvars, ptfvars).Return(true, nil)
```

Comment:
It took me a minute to realize this is mocking the tfexec calls and not the above methods.

Commenting only to point this out in case someone else pauses on the same thing.  Nothing wrong, just the layout of this PR made it a bit confusing.